From 9a200fcedcf81b4fbe833b4a2386bd835265fe93 Mon Sep 17 00:00:00 2001
From: Brian Callahan <ibara@users.noreply.github.com>
Date: Thu, 24 Oct 2019 10:08:45 -0400
Subject: [PATCH 1/9] Remove clear line from example .profile

If you link with ncurses, ^L will clear the screen for you, making this line superfluous.
---
 profile | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/profile b/profile
index 14ea7fe..93ff22e 100644
--- a/profile
+++ b/profile
@@ -6,6 +6,3 @@ export PATH HOME TERM
 
 # Uncomment to set default prompt to current working directory.
 # PS1='$PWD $ '
-
-# Uncomment if you think ksh should act like bash...
-# bind -m '^L'=clear'^J'
-- 
2.25.0

